src/axom/core/docs/sphinx/core_containers.rst

```diff
@@ -241,6 +241,8 @@ via a lambda:
    :end-before: _array_w_raja_end
    :language: C++
 
+.. note:: We need to mark the lambda as `mutable` if we want to modify the array (array view) data.
```

Comment:
Yes, much appreciated.

Reply:
I'm guessing this is because `ArrayView<T>` is copied over as const in a lambda capture? We could probably consider just having a single `T& operator[] const` function in ArrayView - const on an ArrayView<T> should logically apply to the View itself, not the underlying data.